<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ScrollArea" name="ScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1001</width>
        <height>706</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>性能</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="horizontalSpacing">
             <number>110</number>
            </property>
            <property name="verticalSpacing">
             <number>9</number>
            </property>
            <property name="leftMargin">
             <number>30</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>30</number>
            </property>
            <property name="bottomMargin">
             <number>30</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="StrongBodyLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>请不要设置大于自己电脑核心的线程，通常设置为6或者8即可</string>
                </property>
                <property name="statusTip">
                 <string>请不要设置大于自己电脑核心的线程，通常设置为6或者8即可</string>
                </property>
                <property name="whatsThis">
                 <string>请不要设置大于自己电脑核心的线程，通常设置为6或者8即可</string>
                </property>
                <property name="text">
                 <string>开启的线程 jobs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="SpinBox" name="jobs">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>156</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>131</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>32</number>
                </property>
                <property name="value">
                 <number>8</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="CheckBox" name="CBClang">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;速度最快，比mingw64最快能快5倍以上，但是打包报错概率提高，同时你需要自己下载clang编译器，第一次可以尝试，失败后再用mingw64&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="statusTip">
               <string>速度最快，比mingw64最快能快5倍以上，但是打包报错概率提高，同时你需要自己下载clang编译器，第一次可以尝试，失败后再用mingw64</string>
              </property>
              <property name="whatsThis">
               <string>速度最快，比mingw64最快能快5倍以上，但是打包报错概率提高，同时你需要自己下载clang编译器，第一次可以尝试，失败后再用mingw64</string>
              </property>
              <property name="text">
               <string>开启clang
--clang</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="CheckBox" name="CBLowMemory">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>36</height>
               </size>
              </property>
              <property name="toolTip">
               <string>该选项会降低打包速度</string>
              </property>
              <property name="statusTip">
               <string>该选项会降低打包速度</string>
              </property>
              <property name="whatsThis">
               <string>该选项会降低打包速度</string>
              </property>
              <property name="text">
               <string>编译时占用更少的内存
low-memory</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="CheckBox" name="CBMingw64">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>36</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;该选项用来开启mingw64进行加速，nuitka会自动下载&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;该选项用来开启mingw64进行加速，nuitka会自动下载&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>开启mingw64
--mingw64 </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>290</height>
          </size>
         </property>
         <property name="title">
          <string>其他选项</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>180</number>
            </property>
            <property name="verticalSpacing">
             <number>30</number>
            </property>
            <property name="leftMargin">
             <number>30</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>30</number>
            </property>
            <property name="bottomMargin">
             <number>30</number>
            </property>
            <item row="0" column="0">
             <widget class="CheckBox" name="CBRemoveOutput">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>自动删除构建文件夹</string>
              </property>
              <property name="statusTip">
               <string>自动删除构建文件夹</string>
              </property>
              <property name="whatsThis">
               <string>自动删除构建文件夹</string>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>自动删除构建文件夹
remove-output</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="CheckBox" name="CBDisableConsole">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>请不要在一开始直接使用，请确保自己的程序打包之后能够运行再开启这个选项</string>
              </property>
              <property name="statusTip">
               <string>请不要在一开始直接使用，请确保自己的程序打包之后能够运行再开启这个选项</string>
              </property>
              <property name="whatsThis">
               <string>请不要在一开始直接使用，请确保自己的程序打包之后能够运行再开启这个选项</string>
              </property>
              <property name="text">
               <string>关闭命令行
--windows-disable-console</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="CheckBox" name="CBDisableCcache">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>不使用上一次的缓存，防止打包失败</string>
              </property>
              <property name="statusTip">
               <string>不使用上一次的缓存，防止打包失败</string>
              </property>
              <property name="whatsThis">
               <string>不使用上一次的缓存，防止打包失败</string>
              </property>
              <property name="text">
               <string>不使用缓存
disable-ccache</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="CheckBox" name="CBShowMemory">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>显示内存</string>
              </property>
              <property name="statusTip">
               <string>显示内存</string>
              </property>
              <property name="whatsThis">
               <string>显示内存</string>
              </property>
              <property name="text">
               <string>显示内存占用
-show-memory</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="CheckBox" name="CBwindowsUacAdmin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>软件被运行的时候会向用户请求管理员权限运行(仅window)</string>
              </property>
              <property name="statusTip">
               <string>软件被运行的时候会向用户请求管理员权限运行(仅window)</string>
              </property>
              <property name="whatsThis">
               <string>软件被运行的时候会向用户请求管理员权限运行(仅window)</string>
              </property>
              <property name="text">
               <string>软件获取管理员权限
--windows-uac-admin</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="CheckBox" name="CBShowProgress">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>可以在打包的时候显示更多的进度条</string>
              </property>
              <property name="statusTip">
               <string>可以在打包的时候显示更多的进度条</string>
              </property>
              <property name="whatsThis">
               <string>可以在打包的时候显示更多的进度条</string>
              </property>
              <property name="text">
               <string>显示进度条
--show-progress</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="CheckBox" name="CBLto">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭LTO能避免来自编译器的错误，LTO在nuitka是默认开启的&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="statusTip">
               <string>关闭LTO能避免来自编译器的错误，LTO在nuitka是默认开启的</string>
              </property>
              <property name="whatsThis">
               <string>关闭LTO能避免来自编译器的错误，LTO在nuitka是默认开启的</string>
              </property>
              <property name="text">
               <string>关闭lto
--lto=no</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="CheckBox" name="CBQuiet">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>开启安静打包模式</string>
              </property>
              <property name="statusTip">
               <string>该控件有更详细的说明，请点击右上角的小图标再点击这里</string>
              </property>
              <property name="whatsThis">
               <string>该控件有更详细的说明，请点击右上角的小图标再点击这里</string>
              </property>
              <property name="text">
               <string>开启安静打包模式(只输出错误)
--quiet</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="CheckBox" name="CBCleanCache">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>remove-output</string>
              </property>
              <property name="statusTip">
               <string>自动删除构建文件夹</string>
              </property>
              <property name="whatsThis">
               <string>自动删除构建文件夹</string>
              </property>
              <property name="text">
               <string>清除所有缓存
--clean-cache</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>224</height>
          </size>
         </property>
         <property name="title">
          <string>输出exe的信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <property name="horizontalSpacing">
           <number>30</number>
          </property>
          <property name="verticalSpacing">
           <number>15</number>
          </property>
          <item row="2" column="2">
           <widget class="LineEdit" name="LEProductVersion">
            <property name="toolTip">
             <string>产品的版本</string>
            </property>
            <property name="statusTip">
             <string>产品的版本</string>
            </property>
            <property name="whatsThis">
             <string>产品的版本</string>
            </property>
            <property name="placeholderText">
             <string>在这里输入产品的版本,比如 0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="LineEdit" name="LEFileDescription">
            <property name="toolTip">
             <string>产品的描述</string>
            </property>
            <property name="statusTip">
             <string>产品的描述</string>
            </property>
            <property name="whatsThis">
             <string>产品的描述</string>
            </property>
            <property name="accessibleName">
             <string/>
            </property>
            <property name="placeholderText">
             <string>在这里输入产品的描述</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="LineEdit" name="LECompanyName">
            <property name="toolTip">
             <string extracomment="这里设置公司的名字">这里设置公司的名字</string>
            </property>
            <property name="statusTip">
             <string>这里设置公司的名字</string>
            </property>
            <property name="whatsThis">
             <string>这里设置公司的名字</string>
            </property>
            <property name="placeholderText">
             <string>在这里输入公司的名称</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="StrongBodyLabel" name="LBFileVersion">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>文件版本</string>
            </property>
            <property name="statusTip">
             <string>文件版本</string>
            </property>
            <property name="whatsThis">
             <string>文件版本</string>
            </property>
            <property name="text">
             <string>文件版本
--windows-file-version</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="LineEdit" name="LEFileVersion">
            <property name="toolTip">
             <string>文件版本</string>
            </property>
            <property name="statusTip">
             <string>文件版本</string>
            </property>
            <property name="whatsThis">
             <string>文件版本</string>
            </property>
            <property name="placeholderText">
             <string>在这里输入文件的版本,比如 0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="StrongBodyLabel" name="LBCompanyName">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>这里设置公司的名字</string>
            </property>
            <property name="statusTip">
             <string>这里设置公司的名字</string>
            </property>
            <property name="whatsThis">
             <string>这里设置公司的名字</string>
            </property>
            <property name="text">
             <string>公司名称
--windows-company-name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="StrongBodyLabel" name="LBFileDescription">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>产品的描述</string>
            </property>
            <property name="statusTip">
             <string>产品的描述</string>
            </property>
            <property name="whatsThis">
             <string>产品的描述</string>
            </property>
            <property name="text">
             <string>产品描述
--windows-file-description</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="StrongBodyLabel" name="LBProductVersion">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>产品的版本</string>
            </property>
            <property name="statusTip">
             <string>产品的版本</string>
            </property>
            <property name="whatsThis">
             <string>产品的版本</string>
            </property>
            <property name="text">
             <string>产品版本
--windows-product-version</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qmaterialwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qmaterialwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
